3-extra/3-fibonacci.js

```diff
@@ -13,25 +13,29 @@
         - the input n will be passed in to the function as an argument, and will be a number greater than 2
 */
 
-function generateFibonacciSequence(n) {
-    // TODO
-}
+const generateFibonacciSequence = (n) => {
+  const answer = [0, 1];
+  for (let i = 2; i < n; i++) {
+    answer.push(answer[i - 1] + answer[i - 2]);
+  }
+  return answer;
```

Comment:
There are many different algorithms to calculate the Fibonacci sequence, and you might want to experiment with them as you learn different techniques: Recursion, Tabulation, Math, Matrixes, etc. You can see an overview of the possible solutions on [Leetcode ](https://leetcode.com/problems/fibonacci-number/solution/) or [geeksforgeeks](https://www.geeksforgeeks.org/program-for-nth-fibonacci-number/)

Reply:
Hi Alberto, 
Thanks for the comments, I have now adressed these issues.